setup.py

```diff
@@ -17,7 +17,7 @@ def get_test_suite():
     url='https://github.com/Azure/sonic-utilities',
     maintainer='Joe LeVeque',
     maintainer_email='jolevequ@microsoft.com',
-    packages=['config', 'sfputil', 'show', 'sonic_eeprom', 'sonic_installer', 'sonic_psu', 'sonic_sfp', 'acl_loader', 'sonic-utilities-tests', 'clear', 'debug', 'undebug'],
+    packages=['config', 'sfputil', 'show', 'sonic_eeprom', 'sonic_installer', 'sonic_psu', 'sonic_sfp', 'acl_loader', 'sonic-utilities-tests', 'clear', 'debug', 'undebug', 'pfcwd' ],
```

Comment:
This line is getting quite long. Could you please break this definition over multiple lines and also rearrange members alphabetically? E.g.,

```
packages=['acl_loader',
          'clear',
          'config',
          'debug',
          'show',
          ...
          'undebug',
],
```

Reply:
Sure, done